src/components/Vote.jsx

```diff
@@ -0,0 +1,13 @@
+import { useState, useEffect } from "react";
+import { patchVote} from "../../websiteAPI";
+import { Link } from "react-router-dom";
+
+export function addVote(article_id){
+    const [currVote, setCurrVote] = useState(0)
+
+    useEffect(()=>{
+        patchVote(article_id).then(() =>{
+            setCurrVote(currVote + 1)
+        })
+    },[currVote])
+}
```

Comment:
This component does seem to be being used? But I would recommend extracting the logic into a component similar to this but with your logic in `IndividualArticle`.